### ISSUE TITLE
pear_codegen-0.0.17 fails on latest nightly

### ISSUE THREAD

**Issue by RalfJung** [SergioBenitez/Pear#8]
On the current nightly
```
$ rustc --version
rustc 1.28.0-nightly (60efbdead 2018-06-23)
```
the latest pear_codegen fails to build:
```
error[E0023]: this pattern has 6 fields, but the corresponding tuple variant has 4 fields
  --> /home/r/.cargo/registry/src/github.com-1ecc6299db9ec823/pear_codegen-0.0.17/src/lib.rs:66:16
   |
66 |         if let ItemKind::Fn(ref decl, safety, cness, abi, ref generics, ref block) = item.node {
   |                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected 4 fields, found 6

error[E0061]: this function takes 4 parameters but 6 parameters were supplied
  --> /home/r/.cargo/registry/src/github.com-1ecc6299db9ec823/pear_codegen-0.0.17/src/lib.rs:70:24
   |
70 |             let node = ItemKind::Fn(decl.clone(), safety, cness, abi, generics.clone(), new_block);
   |                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected 4 parameters

error: aborting due to 2 previous errors
```

**Comment by SergioBenitez** [comment]
Fixed in v0.0.18. Thanks!

**Comment by RalfJung** [comment]
Thank you for the quick fix. :)
